Framework/Algorithms/inc/MantidAlgorithms/ReflectometrySumInQ.h

```diff
@@ -5,6 +5,8 @@
 #include "MantidAPI/Algorithm.h"
 #include <cmath>
 
+#include <cmath>
```

Comment:
No need to include `cmath` twice. I think this comes from the fact that when we started looking at this, the `#include<cmath>` was actually missing from the Mantid master,  so you (and I also did the same so I could get it to compile in my own branch) added it. But it has since been added to the master (see [here](https://github.com/mantidproject/mantid/commit/949293ad6e5ce329a45a837d05693b0d3c354dfe#diff-d277c3ccd54583fcde61dae4cb0d7fe4) )